Service/appsettings.Development.json

```diff
@@ -6,8 +6,10 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "ConnectionStrings": {
-    "mysql": "Server=localhost; port=3306; database=asesharpdb; user=root; password=asesharp",
-    "airApiKey": "ecc93ce2-d18c-44a3-a414-adc270da84bd"
-  }
+    "ConnectionStrings": {
+        "mysql": "Server=localhost; port=3306; database=asesharpdb; user=root; password=asesharp",
+        //"airApiKey": "ecc93ce2-d18c-44a3-a414-adc270da84bd",
```

Comment:
why comment out not just remove it?

Reply:
Just in case we run out of daily limit calls for internal testing purposes. In the final product, we will only keep one and remove the comment.